src-gen/report.html

```diff
@@ -6,15 +6,15 @@
 		    border-collapse: collapse;
 		    width: 100%;
 		}
-		
```

Comment:
Are the format changes in the report.html file commited intentionally?

Reply:
@JohannaOtt The only changes I made to this file were updates on the related namespace references.

Reply:
@stempler I don't know how these changes were introduced. I am using ATOM.

Reply:
@JohannaOtt We discussed this, and as it is a generated file we cannot generate anyways, we can leave it as it is.